src/app/external-cluster-wizard/template.html

```diff
@@ -57,6 +57,11 @@
             <km-external-cluster-step [formControl]="form.get(stepRegistry.ExternalClusterDetails)"></km-external-cluster-step>
           </ng-container>
         </ng-container>
+
+        <ng-container *ngSwitchCase="stepRegistry.Summary">
+          <km-external-cluster-summary *ngIf="stepperLastIndex"
+                                       [externalClusterSpec]="form.get(stepRegistry.ExternalClusterDetails).value"></km-external-cluster-summary>
```

Comment:
There is no `externalClusterSpec` Input in this component, or am I missing something?